src/Components/WebView/WebView/test/Pages/TestPage.razor

```diff
@@ -0,0 +1,21 @@
+﻿@page "/"
```

Comment:
This is the 'test page' used in the test. It's quite basic, but it is a good end-to-end "is the world not on fire" verification. If this test passes that means that in a WebView we have Blazor loading, serving static files, activating itself, responding to UI input, and updating the rendering. That's pretty good, I think.

Reply:
Test content looks perfect. Agreed this is a good level of detail to cover here!